source/reference/beta-program/orr.txt

```diff
@@ -13,3 +13,53 @@ Enable Oplog Rollover Resilience
    :class: singlecol
 
 .. include:: /includes/beta-feature.rst
+
+.. include:: /includes/orr-intro.rst
+
+By applying changes earlier in the sync process, ``mongosync`` maintains a more 
+recent position in the :term:`oplog`. This adds resilience to long-running 
+operations, mitigates the risk of oplog rollover, and significantly lowers 
+the risk of restarting the sync.
+
+Syntax 
+------
+
+To enable ORR, use the ``--oplogRolloverResilienceIntervalSeconds`` command line 
```

Comment:
(minimalism)
You could omit "the" and "command line option" I think, if you like.